documentation/1-Guides/Arbitraries.md

```diff
@@ -78,6 +78,13 @@ More specific strings:
 - `fc.webUrl()` Web URL following the specs specified by RFC 3986 and WHATWG URL Standard
 - `fc.emailAddress()` Email address following RFC 1123 and RFC 5322
 
+## Date (:date)

```

Comment:
Date (:Date) - the text after ":" is supposed to be the type of data produced by the arbitrary